### DISCUSSION TITLE
在 xmake.lua 中，怎么获取当前的 LUA 版本

### DISCUSSION THREAD

**Discussion by aceyin** [xmake-io/xmake#2354] [Q&A] [answered]
因为需要根据LUA的版本，来做一些前置判断，请问一下：在 xmake.lua 中，怎么获取当前的 LUA 版本？

**Comment by waruqi** [comment, marked as answer]
目前取不了 lua 版本，只能通过 `xmake.luajit()` 判断是否为 luajit。。。如果是 luajit 就是  5.1 ，否则就是 lua 5.4

**Reply by aceyin** [reply]
谢谢~~
那就只能 os.exec("lua -v") 来绕一绕了~~

**Reply by waruqi** [reply]
没必要

**Comment by aceyin** [comment]
按照我初步理解，xmake 自己内置了 LUA 解析器，自己有自己的LUA运行环境，并没有依赖操作系统环境中的LUA。
对吗？

**Reply by waruqi** [reply]
嗯

**Reply by aceyin** [reply]
明白~~
感谢！
xmake不错，正研究怎么用来放在 LUA + C 的项目里面使用

加油~~
